libbroker/broker/event.hh

```diff
@@ -73,6 +81,29 @@ public:
       description(std::move(description)) {}
 };
 
+constexpr event::component_mask operator|(event::component_type lhs,
+                                          event::component_type rhs) noexcept {
+  auto res = static_cast<uint32_t>(lhs) | static_cast<uint32_t>(rhs);
+  return static_cast<event::component_mask>(res);
+}
+
+constexpr event::component_mask operator|(event::component_mask lhs,
+                                          event::component_type rhs) noexcept {
+  auto res = static_cast<uint32_t>(lhs) | static_cast<uint32_t>(rhs);
+  return static_cast<event::component_mask>(res);
+}
+
+constexpr event::component_mask operator|(event::component_type lhs,
+                                          event::component_mask rhs) noexcept {
+  auto res = static_cast<uint32_t>(lhs) | static_cast<uint32_t>(rhs);
+  return static_cast<event::component_mask>(res);
+}
```

Comment:
No need to do, but to disable just a single component from the "default mask", this would miss some `&` and `~` operators?

Reply:
Right. I didn't have a use case yet, so I've skipped them for now.